src/ttsim/interface_dag_elements/data_converters.py

```diff
@@ -3,6 +3,7 @@
 from typing import TYPE_CHECKING, Any
 
 import dags.tree as dt
+import numpy as np
```

Comment:
```suggestion
import numpy
```

This is the convention we use in GETTSIM in order to be very clear what we are using.